lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -281,6 +284,19 @@ def test_unicode_handling_in_url(self):
             response = self.client.get(request_url)
             self.assertEqual(response.status_code, 404)
 
+    def test_syllabus(self):
```

Comment:
Separate out the test cases here. If the first test were to fail, we wouldn't know if the other 2 failed.